stdlib/sqlite3/dbapi2.pyi

```diff
@@ -132,7 +137,7 @@ class Connection(object):
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     def close(self) -> None: ...
     def commit(self) -> None: ...
-    def create_aggregate(self, name: str, num_params: int, aggregate_class: type) -> None: ...
+    def create_aggregate(self, name: str, num_params: int, aggregate_class: Type[_AggregateProtocol]) -> None: ...
```

Comment:
I haven't looked at this, but could this also be `Callable[[], _AggregateProtocol]`?

Reply:
They imply in the docs and argument name that it should be a class...
But probably not. Eventually it's being called with https://github.com/python/cpython/blob/e9194ea6eaa18299d6ccbd3555ce150fab0c6184/Modules/_sqlite/connection.c#L659

so I think I can this to your suggestion.